docs/plugins/repository-s3.asciidoc

```diff
@@ -447,3 +447,9 @@ limitations.
 
 Instances residing in a public subnet in an AWS VPC will connect to S3 via the
 VPC's internet gateway and not be bandwidth limited by the VPC's NAT instance.
+
+Note that Amazon has published instructions at 
+https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc-endpoint.html
```

Comment:
I think it would be better to link to the [Gateway VPC endpoints](https://docs.aws.amazon.com/vpc/latest/userguide/vpce-gateway.html) docs rather than linking to the CLI docs, wdyt @vocatan?

Reply:
You're probably right - I tend to go straight for the jugular and give the CLI equivalent where possible, given that configuring ElasticSearch has largely been a CLI exercise for me in the past.

Reply:
I think it would be better if we change the reference to the general documentation, so we cover all cases. wdyt 
@vocatan ?